### PR TITLE
Reduces Tourette Stun Time

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
tweak: Tourette's stun time has been reduced from 20 seconds to 10 seconds.
/:cl:

Tourette is extremely annoying and has a relatively high chance of appearing in just 30 seconds, because each roll is made per tick. This means you can repeatedly get stunlocked for more than 2 minutes if you're unlucky, completely helpless as you automatically swear on the radio.

### CODE REVIEW

**Comment by ghost** [comment]
You currently have a negative Fix/Feature pull request delta of -1. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by ghost** [comment]
lets make a compromise with the current time and make it 10 seconds to make it require some sort of external help instead of powering through it unlike autism

**Comment by Arzion5** [comment]
Sure

**Comment by ghost** [comment]
sweet you got my +1

File: hippiestation/code/datums/mutations.dm [hunk 1]
```diff
@@ -54,4 +54,8 @@
 	regenerate_icons()
 	visible_message("<span class='danger'>[src]'s body glows green, the glow dissipating only to leave behind a cluwne formerly known as [src]!</span>", \
 					"<span class='danger'>Your brain feels like it's being torn apart, and after a short while, you notice that you've become a cluwne!</span>")
-	flash_act()
\ No newline at end of file
+	flash_act()
+
+/datum/mutation/human/tourettes/on_life(mob/living/carbon/human/owner)
```
Comment (JohnGinnane) [inline hippiestation/code/datums/mutations.dm:59]: This only stuns and doesn't do any of the following: https://github.com/HippieStation/HippieStation/blob/229f7c8efab323a4b69e58233e16d75aa75cce2d/code/datums/mutations.dm#L322

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

**Review by JohnGinnane — APPROVED** [review]
Code is fine, but I still think 10 seconds is fucking extreme

**Comment by JohnGinnane** [comment]
Oh, I think you should include your name after the first :cl: so it appears in the change log

e.g. #4710

**Comment by Arzion5** [comment]
The name of the coder automatically appears in a changelog

**Comment by JohnGinnane** [comment]
Oh cool, wasn't sure

**Comment by dreamcrab** [comment]
was it always like this? this seems like a wild stun at 20 seconds jfc

+31412

**Comment by PopNotes** [comment]
I think it'd be better if the stun time was only five seconds to make epilepsy even scarier by comparison.

**Comment by KayeArray** [comment]
yeah it's pretty gay +1

**Comment by ghost** [comment]
Your Fix/Feature pull request delta is currently below zero (-2). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**